DirectEmployers/MyJobs
# [PD-3883]  calendar updates should not clear search

## What's this PR do?
Prevent search term from being cleared when the calendar is updated. 

## How should this be manually tested?
compare search box behavior on QC and this branch, notice how it's affected by selecting new dates from the calendar.

## Any background context you want to provide?
Added an action to set the search term ('partial') for the currently active tab. This search value is then applied on the backend when the user updates the calendar.

## PR Checklist

### Front-End
